lang/rustup/rustc-triple.mk

```diff
@@ -0,0 +1,76 @@
+# To enable rustc/cargo support in your packages, add:
+# include $(TOPDIR)/feeds/packages/lang/rustup/rustc-triple.mk
```

Comment:
What if packages feed is named differently?

Reply:
This is something I seriously struggled with, since I don't know another way to ultimately pass $(RUSTC_TARGET_ARCH) to the packages that need the rust variables. What is the best way to do this?

Reply:
There is a similar dilemma with the Python include files ([Using Python in external/other package feeds](https://github.com/openwrt/packages/tree/master/lang/python#using-python-in-externalother-package-feeds)). For packages in this repo, relative paths are best ([Include python3-package.mk](https://github.com/openwrt/packages/tree/master/lang/python#include-python3-packagemk)).

Reply:
@jefferyto What about if (example) ./net/suricata5 needed to call it?  Just something like:

include ../../lang/rustc-triple.mk?

Reply:
Yes that should work.